static/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -71,12 +70,12 @@ const upcomingTasksHeading = (
 );
 const completedTasksHeading = <Heading key="complete">{t('Completed')}</Heading>;
 
-export const useGetTasks = (
+export const useOnboardingTasks = (
```

Comment:
useOnboardingTasks right now is only returning the most recently completed tour as complete if I go through the tours, unless I refresh the page. For example, I complete the Sidebar tour, it'll show up as complete. Then I complete the issues tour, and then it will only show issues as complete. I refresh the page and then they both show up as complete

Reply:
fixed!